src/OWML.Logging/ConsoleUtils.cs

```diff
@@ -17,15 +17,14 @@ public static void WriteByType(MessageType type, string line)
 				MessageType.Error => ConsoleColor.Red,
 				MessageType.Warning => ConsoleColor.Yellow,
 				MessageType.Success => ConsoleColor.Green,
-				MessageType.Message => ConsoleColor.Gray,
+				MessageType.Message => ConsoleColor.White,
 				MessageType.Info => ConsoleColor.Cyan,
 				MessageType.Fatal => ConsoleColor.Magenta,
-				MessageType.Debug => ConsoleColor.Blue,
-				_ => ConsoleColor.Gray
+				MessageType.Debug => ConsoleColor.Gray,
```

Comment:
what about having debug warnings come up in yellow? would make it easier to distinguish between debug logs and debug warnings

Reply:
Yellow reminds me of warnings...

Reply:
![image](https://user-images.githubusercontent.com/3322367/103079639-80219580-45d4-11eb-8497-e8eb14fe973f.png)
Dark gray?

Reply:
Pushed.

Reply:
Ah wait, didn't read your comment properly. Debug *warnings*... you mean from Unity?

Reply:
yeah

Reply:
light gray is fine for debug _logs_, but for debug _warnings_ it should maybe be dark yellow?

Reply:
Unity warnings now have same color as "regular" warnings, but are still only shown in debug mode. For another color we'd have to introduce another message type 🤷